mep/people/views.py

```diff
@@ -244,6 +249,29 @@ def get_context_data(self, **kwargs):
                 'endDate': end.isoformat()
             } for start, end in account.event_date_ranges]
         }
+
+        # plottable locations for member address map visualization
+        # addresses can be stored on either Person or Account
```

Comment:
I thought we were just going to use addresses on account, to keep it simpler? 

Seems like we ought to be able to get this directly from the person, but maybe not until we are able to modify the models so that people can only have one account. Something like: `person.account_set.all().first().address_set.all()`